src/platforms/react-native/manual-setup/hermes.mdx

```diff
@@ -24,30 +24,38 @@ If you are using Hermes, you should not have the [RAM bundles](/platforms/react-
 
 ## Source Maps
 
-If you do not need custom sourcemaps, the `sentry.gradle` build step fully supports Hermes source maps.
+If you do not need custom source maps, the `sentry.gradle` build step fully supports Hermes source maps.
 
 ## Custom Source Maps
 
-If you are manually bundling and building React Native apps, follow this three-step process.
+To manually generate source maps follow this three-step process. Be aware that it's important to exactly match the steps of your React Native tooling. Otherwise the generated bundle and source maps might be working incorrectly, for example pointing to unrelated lines. The following command are examples of the default flow.
```

Comment:
`follow this three-step process.` this reads a bit confusing to me. 
First, a reference to match them exactly with my tooling. Then another header `Compile Source maps` followed by a reference to `not all the steps are required.`?

Reply:
Could this be a generic list of steps instead?
i.e.
- Do something
- (if not `EXTRA_PACKAGER_ARGS` on `iOS`) do the other thing
- ...
- profit!

Reply:
I see, I'll try to rephrase it.